README.md

```diff
@@ -732,14 +732,14 @@ As you can see all the classes in a class hierarchy actually share oneclass vari
 
 - Use `Set` instead of `Array` when dealing with unique elements. `Set` implements a collection of unordered values with no duplicates. This is a hybrid of `Array`'s intuitive inter-operation facilities and `Hash`'s fast lookup.
 
-- Use symbols instead of strings as hash keys where possible.
+- Use symbols instead of strings as hash keys where possible, and use the Ruby 1.9 hash syntax rather than hash rockets.
```

Comment:
Should this read as `Use symbols instead of strings as hash keys and use the Ruby 1.9 hash syntax rather than hash rockets where possible.`?